mail-filter/spamassassin/spamassassin-3.4.3.ebuild

```diff
@@ -0,0 +1,284 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit perl-functions systemd toolchain-funcs user eapi7-ver
```

Comment:
... and if EAPI-7 works, you can remove `eapi7-ver` here. I didn't see it being used anyway in this ebuild.